library/Fum2GitHub/GitHub.hs

```diff
@@ -0,0 +1,90 @@
+{-# LANGUAGE OverloadedStrings #-}
+module Fum2GitHub.GitHub (
+    getNextUrl,
+    getOrgMembers,
+) where
+
+import           Data.Aeson ((.:))
+import qualified Data.Aeson as Aeson
+import           Data.Aeson.Types (parseEither)
+import qualified Data.ByteString.Lazy as LBS
+import qualified Data.Text as T
+import qualified Data.Text.Encoding as E
+import           Data.Traversable (traverse)
+import qualified Data.Vector as Vector
+import           Network.HTTP.Conduit (
+    applyBasicAuth,
+    httpLbs,
+    parseUrl,
+    requestHeaders,
+    responseBody,
+    responseHeaders,
+    withManager)
+import           Network.HTTP.Types.Header (ResponseHeaders)
+import           Text.Regex (mkRegex, matchRegex)
+
+
+-- Get the response body and headers from url using oAuthToken.
+getHttp :: String -> String -> IO (LBS.ByteString, ResponseHeaders)
```

Comment:
would add `type URL = String` and `type OAuthToken = String` for better documentation.
And using `newtype` would prevent to accidentally swapping the params.